src/pages/index.js

```diff
@@ -26,6 +29,49 @@ const IndexPage = () => {
             </Link>
           </div>
         </div>
+        <div className="my-24 lg:my-56">
+          <div className="grid mx-auto max-w-lg text-center mb-10">
+            <p className="text-3xl mb-5 font-extrabold">My work</p>
+            <p className="text-violet-light">
+              Here is a different subheading text explaining this section. Well,
+              it can certainly take its time to make a point. It's okay.
+            </p>
+          </div>
+          <div className="uppercase font-medium tracking-widest relative text-xs text-violet-light text-opacity-60 text-center mb-5">
+            <p className="relative inline-block bg-gray-800 z-10 px-4">
+              Choose category
+            </p>
+            <div className="absolute w-full h-px border-gray-700 border-b inset-y-0 my-auto" />
+          </div>
+          <div className="flex justify-center mb-10">
+            <Link to="#">
+              <button className="bg-gradient-to-b focus:from-violet focus:to-violet-dark px-6 rounded-full w-max">
+                3D
+              </button>
+            </Link>
+            <Link to="#">
+              <button className="bg-gradient-to-b focus:from-violet focus:to-violet-dark px-6 rounded-full w-max">
+                Drawing
+              </button>
+            </Link>
+            <Link to="#">
+              <button className="bg-gradient-to-b focus:from-violet focus:to-violet-dark px-6 rounded-full w-max">
+                Painting
+              </button>
+            </Link>
+          </div>
+          <div className="container grid md:grid-cols-2 lg:grid-cols-3 gap-10 bg-gray-900 p-10 rounded-2xl bg-opacity-40">
+            <div className="shadow-xl rounded-lg overflow-hidden aspect-w-1 aspect-h-1">
+              <img src={Sofa} className="object-cover" />
+            </div>
+            <div className="shadow-xl rounded-lg overflow-hidden aspect-w-1 aspect-h-1">
+              <img src={Sofa} className="object-cover" />
+            </div>
+            <div className="shadow-xl rounded-lg overflow-hidden aspect-w-1 aspect-h-1">
+              <img src={Sofa} className="object-cover" />
```

Comment:
Don't forget that each image should have an `alt` prop.

Reply:
Done!